scripts/start.js

```diff
@@ -64,6 +64,8 @@ const localhost = '0.0.0.0';
 
   const devServer = new WebpackDevServer(parentCompiler, {
     contentBase: paths.public,
+    publicPath: paths.publicPath,
+    host: appHosts[0],
```

Comment:
Just double checking this doesn't impact using the other hosts?

Reply:
Any example you can think of? Happy to test further.